### ISSUE TITLE
Footer: English text on French pages

### ISSUE THREAD

**Issue by mmiron-HC** [mmiron-HC/WST-RSAMS#61]
URL: all footers
User Agent: Chrome
Language: French

**Issue Description:** English text on French page
Symbol of Government of Canada alternative text is in EN on the FR page.
Hidden Heading "About this site" (which is the wrong heading) is in EN on the French page
Top of the page text is in EN on the French page. 

**Steps to reproduce:**
View page, turn off CSS (Styles - run WAVE, turn styles off)
navigate to the footer
![image](https://user-images.githubusercontent.com/65728700/129063364-de301ce6-8d0c-4d10-a039-3ce841c1e4c5.png)

**Expected results:**
all text on the French page should be in French.

**Comment by ndolter** [comment]
Validated that the French page now displays the following:
- Alt text = Symbole du gouvernement du Canada
- Au sujet du gouvernement
- Haut de la page
